docs/technical-documentation/Security_Assessment.md

```diff
@@ -0,0 +1,101 @@
+# Security Assessment Policy-Hub 
+
+|                           |                                                                                                |
+| ------------------------- | ---------------------------------------------------------------------------------------------- |
+| Contact for product       | [@evegufy](https://github.com/evegufy) <br> [@jjeroch](https://github.com/jjeroch)             |
+| Security responsible      | [Szymon Kowalczyk](szymon.kowalczyk@zf.com) |
+| Version number of product |                                                                                           |
+| Dates of assessment       | 2024-02-16: Assessment                                                                      |
+| Status of assessment      | ASSESSMENT DRAFT                                                                            |
+
+
+## Product Description
+
+Policy-Hub project is an readonly REST API project, without implementation of an UI. (Pure Backend Component)
+
+Main purpose of the product is to provide interested and authenticated C-X Users with read access to policy data, use cases, credential types, policy rules, that are created during an application seeding.
+
+The Policy-Hub comprise the technical foundation for interaction, monitoring, auditing and further functionalities. 
+They are state of the art in terms of technology portfolio, consist of open-source components whenever possible and are open-sourced themselves 100%.
+
+Policy-Hub can be run anywhere: it can be deployed as a docker image, e. g. on Kubernetes (platform-independent, cloud, on prem or local).
+
+The policy hub is using following key frameworks:
+
+- .Net
+- Entity Framework
+
+Please find more details about architecture & development concept- https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/architecture/Development%20Concept.md
+
+### Important Links
+
+Latest released version: 0.1.0-rc.3
```

Comment:
it's not a link and could you please change to 0.1.0 without postfix? as that's the version we're currently wrapping up with release candidates